feisty/offline_driver.py

```diff
@@ -110,6 +109,10 @@ def __init__(
         settings_in={},
         fish_ic_data=None,
         benthic_prey_ic_data=None,
+        biomass_init='constant',
+        allow_negative_forcing=False,
+        diagnostic_names=[],
```

Comment:
do we want to provide a utility to query available diags? Or have a minimal set of defaults?